src/Controller.ts

```diff
@@ -66,8 +66,13 @@ export const Controller = {
 
 	getEnabledEngines: async (): Promise<RuleEngine[]> => {
 		const allEngines: RuleEngine[] = await Controller.getAllEngines();
-		const engines: RuleEngine[] = allEngines.filter(e => e.isEnabled());
+		const engines: RuleEngine[] = [];
```

Comment:
Array#filter doesn't work with promises. Broke out into a for loop.

Reply:
If you want a one-line fix, I think this should work:
`(await Promise.all(allEngines.map(e => await e.isEnabled() && e))).filter(e => e != null)`